sql/moz-fx-data-shared-prod/internet_outages/mobile_global_outages_v1/query.sql

```diff
@@ -0,0 +1,180 @@
+-- combine baseline pings from different products
+
+with
+-- Firefox for Android
+fenix_baseline as (
+    select
+        metadata.geo.country,
```

Comment:
Should we just use your udf here and have `??` if values are missing. This way we know that downstream we only need to deal with `??`

Reply:
thought about it, but since we are not joining we don't really need it though?